doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -59,13 +59,31 @@
-> * Network Service Mesh related considerations
-> * Number of interfaces and IP addresses to be supported by a pod
-> * IPv6 single and dual stack needs
+Because `req.inf.ntw.01` requires the architecture to support CNI and `req.inf.ntw.16` requires the capability to attach several network interfaces to the pods the architecture must support a CNI metaplugin/CNI multiplexer.
+
+A CNI metaplugin/CNI multiplexer is capable to attach several interfaces, using different other CNI plugins, to a pod. The different network charastheristics of the interfaces, which require different networking technologies require different CNI plugins.
+
+Inter node communication required by `req.inf.ntw.02` must be served by a CNI plugin with basic networking capabilities, like Calico or Flannel.
+
+There are two types of low latency and high throughput networks required by `req.inf.ntw.04`. Network used for signalling traffic are more demanding than what an overlay network can handle, but still does not need the usage of user space networking. Due to the nature of signalling protocols used this type of networks require the NAT-less communication stated by `req.inf.ntw.03`. Due to the combination of these two requirements networks with this characteristics must be served by a CNI plugin with IPVLAN or MACVLAN support.
+
+The low latency, high throughput networks for handling the user plane traffic require the capability to use an user space networking technology. In case of DPDK the usage of SR-IOV VF-s is recommended.
+
+As `req.inf.ntw.14` mandates the architecture must enable the integration of different SDN solutions via their respective CNI integration.
+
+The architecture must support networking for telecom equipments in an environment where the networks of the CNF-s are set up by the network adminisrators of the telecom operator. This is why, as `req.inf.ntw.10` requires, the architecture must provide a set of abstract management API-s to manage the network connectivity of the CNF pods.
+The API must support multiple tenants and must require elevated acces rights to manipulate infrastructure related API objects as these operations require reconfiguration of the physical network infrastructure.
+
+To fullfill the requirements of `req.inf.acc.02` the architecture must support the usage of device plugins via the Device Plugin API, also the alignment of the devices and the CPU topology must be supported using the [Topology Manager](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/).
+
+The architecture must support both IPv4, IPv6 and dual stack interfaces of the workloads.
+ > How about IPv6 support of the control plane?
+
+As Kubernetes Ingress and Services have no support for all the protocols needed in telecommunication environments (Diameter, SIP, LDAP, etc) and their capacity is limited the architecture must enable the usage of alternative load balancers, like external or built into the application.
```

Comment:
Also do we want to take a stance on Ingress / Egress / Services?
Option1: we don't need them
Option2: We need them, but we require the architecture to support them only on the "cluster network"
Option3: We need them, and we require the architecture to make them work over all interfaces

Reply:
I have no strong opinion on this. Any suggestions?

Reply:
I think we definitely need to define that Services are needed for inter-Pod communications, for sure.  For incoming communication, I think we need to avoid describing "how" something is deployed (i.e. "you must use Ingress" or "you must use LoadBalancer Service type", etc.).  I think it could be fair to say something like @CsatariGergely already has, but adding that those external load balancers must be managed via Kubernetes API objects (e.g. cloud-controller for LoadBalancer Services, or CRDs, etc.). That way we're specifying an interface that can be tested against.
What do you think?

Reply:
In some cases the telco protocols are still used in inter-Pod communication. A load balancer is a typical example when the LB pod only decides the next hop, but does not do any protocol translation. 
Managing external LB-s via Kubernetes API objects I will add.

Reply:
agree with Gergely. it is a far fetched assumption IMO that all CNF internal communication is stateless, and happens in L7